src/pages/gen2/build-a-backend/add-aws-services/custom-resources/index.mdx

```diff
@@ -13,13 +13,17 @@ export function getStaticProps(context) {
 
 }
 
----
-title: Custom Resources
----
+Custom resources can be written using the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/api/v2/).
 
-@TODO better examples of constructs and stacks
+From the [Concepts - AWS Cloud Development Kit (AWS CDK) v2](https://docs.aws.amazon.com/cdk/v2/guide/core_concepts.html) documentation
 
-Custom resources can be written using the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/api/v2/). You have the option to create individual [Constructs](https://docs.aws.amazon.com/cdk/v2/guide/constructs.html) or [Stacks](https://docs.aws.amazon.com/cdk/v2/guide/stacks.html).
+:::info
```

Comment:
this (and other references like this) are from [docusaurus's admonitions](https://docusaurus.io/docs/markdown-features/admonitions), whic unfortunately do not exist in this docs site. Can we port this to a `<Callout>`?